components/create_post/create_post.jsx

```diff
@@ -953,7 +953,12 @@ export default class CreatePost extends React.Component {
         let emojiPicker = null;
         if (this.props.enableEmojiPicker && !readOnlyChannel) {
             emojiPicker = (
-                <span className='emoji-picker__container'>
+                <span
+                    role='button'
+                    tabIndex='0'
+                    aria-label='Click to open emojies popup'
```

Comment:
I think we're using emojis as plural form.

Also, in this context, we may use `Click to open emoji picker`